test/unit/mdl-checkbox/foundation.test.js

```diff
@@ -18,6 +18,9 @@ import test from 'tape';
 import bel from 'bel';
 import lolex from 'lolex';
 import td from 'testdouble';
+
+import {setupFoundationTest} from '../helpers/setup';
```

Comment:
Yeah similar to above I figured I'd tackle some refactors along with this PR so that we can minimize tech debt while pushing out features at the same time. 

Should I split these cleanup changes into a separate commit and then we could merge the PR in without squashing the commits?